### PR TITLE
Fixes ERP in the station names.

### PR DESCRIPTION
**Pull request by Iamgoofball**
rule 8 violation

### CODE REVIEW

**Comment by realthomas69** [comment]
controversial

**Comment by Cheshify** [comment]
based

**Comment by tralezab** [comment]
notears

File: strings/station_names.txt [hunk 1]
```diff
@@ -26,7 +26,6 @@ Slant-ways
 Widdershins
 Rimward
 Expensive
-Procreatory
 Imperial
 Unidentified
 Immoral
```
Comment (Kylerace) [inline strings/station_names.txt:31]: bro and you left this in? smh

**Pull request closed without merging**